### DISCUSSION TITLE
How do I implement fonts to my Next app?

### DISCUSSION THREAD

**Discussion by NikRoe** [neuefische/questions#153] [Web] [answered]
Hey zusammen,

ich arbeite das erste Mal mit dem tollsten Framework der Welt: NextJS. Ich habe leider ein paar Probleme beim Einbinden von Fonts.

In der [Dokumentation](https://nextjs.org/docs/basic-features/font-optimization) habe ich gesehen, dass ich etwas in der `_document.js` einbinden muss.

Ich habe in der `_document.js `schon ein paar Dinge hinzugefügt, aber leider klappt es noch nicht.
Ein Screenshot meiner `_document.js`:

Ich habe alle Imports zusammengefasst:

![Screenshot 2022-09-21 at 10 46 23](https://user-images.githubusercontent.com/98757391/191459163-5df67d24-cfd5-4a27-b541-a18d22bed650.png)
![Screenshot 2022-09-21 at 10 47 40](https://user-images.githubusercontent.com/98757391/191459402-9f2598f7-6863-4d23-9ff8-4d80431bead7.png)

Wenn ich über die Fehlermeldung gehe, wird mir dies angezeigt:
![Screenshot 2022-09-21 at 10 49 33](https://user-images.githubusercontent.com/98757391/191459814-0e22131c-277f-4e21-8932-9ba6d4e1d223.png)

Einen [Pull Request](https://github.com/NikRoe/next-js-intro/pull/2) habe ich ebenfalls erstellt.

Könnt ihr mir dabei helfen?

**Comment by ThomasKirst** [comment, marked as answer]
Hi Niklas,

good try, but I identified two issues:

**First, check your import on line 1 – it should be**:

```js
import Document, { Html, Head, Main, NextScript } from 'next/document'
```

`Document` is the default export from `next/document`.

**Second, since this is a JS class we cannot simply add the return statement without an enclosing method.**
With class based components, we always need a `render()` method which wraps the `return` statement.

```jsx
render() {
  return (
    <Html>
      <Head>
        <link
          href="https://fonts.googleapis.com/css2?family=Inter&display=optional"
          rel="stylesheet"
        />
      </Head>
      <body>
        <Main />
        <NextScript />
      </body>
    </Html>
  )
}
```

This should do the trick ;)

**Reply by NikRoe** [reply]
Hey Thomas, 

now it works, thanks a lot :)
